internal/pkg/api/client/storageapi/token_test.go

```diff
@@ -30,22 +30,22 @@ func TestGetToken(t *testing.T) {
 	t.Parallel()
 	project := testproject.GetTestProject(t, env.Empty())
 	logger := log.NewDebugLogger()
-	api := New(context.Background(), logger, project.StorageApiHost(), false)
+	api := New(context.Background(), logger, project.StorageAPIHost(), false)
```

Comment:
Acronyms sa v Go pisu vzdy velkym, v tomto projekte sme to neriesili, ale v API klientovi je to uz spravne (kontroluje to aj linter):
`The rationale for uppercase abbreviations in Go symbol name derives from the english convention for writing acronyms using all uppercase letters`